Atomic/syscontainers.py

```diff
@@ -2628,21 +2647,15 @@ def container_exec(self, name, detach, args):
         else:
             checkout = self._canonicalize_location(self.get_checkout(name))
             if checkout is None:
-                raise ValueError("The container '{}' doesn't exist".format(name))
+                return self._run_once(name, "run-once-{}".format(os.getpid()), args=args)
```

Comment:
> https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_version/tasks/first_master_containerized_version.yml#L22

Wrong line? This is what I saw: 
```
when:
  - openshift_version is not defined
- not openshift_use_crio_only
```
Might not be relevant but just a nit.

Reply:
I changed the logic of this code as I was not really happy about it.

Now it attempts to run a new container if the specified NAME is a fully qualified name.